code/modules/jobs/job_types/roguetown/youngfolk/vagabond.dm

```diff
@@ -6,7 +6,7 @@
 	allowed_races = RACES_ALL_KINDS
 
 	advclass_cat_rolls = list(CTAG_VAGABOND = 20)
-	wanderer_examine = TRUE
+	wanderer_examine = FALSE
```

Comment:
False by default. Remove this line and give it another go. You'll see it works identically. No need to go false, when that's the normal.